core/src/components/range/range.tsx

```diff
@@ -325,7 +325,8 @@ export class Range implements ComponentInterface {
       this.setupGesture();
     }
 
-    this.contentEl = findClosestIonContent(this.el);
+    const ionContent = findClosestIonContent(this.el);
```

Comment:
The scroll target test was failing because we didn't take into account that there might be a custom scroll target.